fix: Fix pathRewrite in k8s Proxy for WebSocket requests (#18716)
backstage/backstage: .changeset/chatty-seahorses-juggle.md

---
'@backstage/plugin-kubernetes-backend': patch
---

> WebSocket requests path were not being rewritten by the proxy properly, now they do.

Reviewer: Not blocking, I just kinda like the imperative mood for release notes:
```suggestion
Fixed a bug where the proxy was not rewriting WebSocket request paths properly.
```
